changelog/pending/20230426--sdk-nodejs--support-loading-package-json-from-parent-directory.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: sdk/nodejs
+  description: Support loading package.json from parent directory. If `package.json` is not found in the current directory, Pulumi recursively searches up the directory tree until it is found. If `package.json` provides a `main` field, that field is relative to the current directory.
```

Comment:
I feel "Relative to the current directory" isn't precise enough. 
Relative to the directory the package.json was found?
Relative to the current working directory pulumi was started in?
Relative to the project root (i.e. where Pulumi.yaml was found)?
Relative to the project main (i.e. where "main" in the Pulumi.yaml pointed)?

Reply:
I addresses this by invoking the terminology you've used elsewhere.